R/eval-relocate.R

```diff
@@ -161,3 +169,15 @@ eval_relocate <- function(expr,
 
   sel
 }
+
+with_rename_errors <- function(expr, arg, error_call) {
+  tryCatch(
```

Comment:
```suggestion
  try_fetch(
```

To get a more complete full backtrace and a more useful `recover()`.

Reply:
Do you want me to also change to `try_fetch()` in `with_subscript_errors()` then? I'll do it in another PR

Reply:
Sounds good, thanks